labs/computing-experiment-metrics/README.md

```diff
@@ -1,11 +1,12 @@
-# Computing Experiment Datasets #1: Experiment Subjects
+# Optimizely Labs Template Notebook
 
-This Lab is part of a multi-part series focused on computing useful experiment datasets. In this Lab, we'll use [PySpark](https://spark.apache.org/docs/latest/api/python/index.html) to compute _experiment subjects_ from [Optimizely Enriched Event](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-export) ["Decision"](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-data-specification#decisions-2) data.
+This Lab serves as a template for building [Optimizely Labs](https://www.optimizely.com/labs) using Jupyter notebooks.  It contains
+- a template notebook for lab content (this document)
```

Comment:
Should we specify that this is `data lab` content or `Jupyter notebook` content? Just want to make it clear that not all labs use `Jupyter`

Reply:
See above; the template language shouldn't be in this lab..